app/assets/stylesheets/bootstrap_overrides.less

```diff
@@ -33,16 +33,102 @@ body {
   height: auto !important;
   height: 100%;
   /* Negative indent footer by it's height */
-  margin: 0 auto -@baseFontSize*1.5;
+  margin: 0 auto -@font-size-base*1.5;
 }
 
+.page-header{
+  margin: 20px 0 30px;
+}
+
+.form-report{ 
+  .input-sm {
+    width: 220px;
+   }
+
+   label{
+    font-weight: normal;
+   } 
+}
```

Comment:
It seems to me that the overwrites for page-header and form-report don't have a big effect. The only thing I notice when removing them is that the forms on the event page and elsewhere move a bit up.

If this is the only thing they touch, I would remove them.